src/main/java/org/openhmis/manager/ExitManager.java

```diff
@@ -298,4 +297,4 @@ public static TmpExit generateTmpExit(ExitDTO inputDTO) {
 		return tmpExit;
 	}
 	
```

Comment:
This file has `getExits(updateDate)` and `getExitByEnrollmentId` which are both obsolete and should be removed.